libs/designsystem/src/lib/components/button/button.component.ts

```diff
@@ -86,8 +86,11 @@ export class ButtonComponent implements AfterContentInit {
     }
   }
 
-  private findNonCommentSibling(nativeElement: any, direction: 'nextSibling' | 'previousSibling') {
-    const sibling = nativeElement[direction];
+  private findNonCommentSibling(
+    nativeElement: HTMLElement,
+    direction: 'nextSibling' | 'previousSibling'
+  ) {
+    const sibling = nativeElement[direction] as HTMLElement;
```

Comment:
Consider using direction to conditionally get sibling instead:
```suggestion
    direction: 'previous' | 'next'
  ) {
    const sibling = direction === 'previous' ? nativeElement.previousSibling : nativeElement.nextSibling;
```

Reply:
Should be resolved by new commits 😊